feature/metis/manage-conversations/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/manageconversations/ui/conversation/overview/ConversationOverviewBody.kt

```diff
@@ -168,6 +192,73 @@ fun ConversationOverviewBody(
     }
 }
 
+@Composable
+fun ConversationFabMenu(
+    onCreateChat: () -> Unit,
+    onBrowseChannels: () -> Unit,
+    onCreateChannel: () -> Unit,
+    canCreateChannel: Boolean
```

Comment:
Nit: I think the code is more readable when you put lambda arguments at the end.